src/source/Signaling/LwsApiCalls.c

```diff
@@ -643,6 +643,10 @@ STATUS lwsCompleteSync(PLwsCallInfo pCallInfo)
     return retStatus;
 }
 
+BOOL isCallResultFailureRetryable(PCallInfo pCallInfo) {
```

Comment:
I don't think there is an established spelling for this since neither is actually a dictionary word.  `Retryable` looks/feels more correct to me, but I will change it if you insist :).